# the rotator angle (deg)
uint16 rotation
uint8[192] data